userservice/src/main/java/com/gymbro/userservice/controller/AuthenticationController.java

```diff
@@ -22,7 +22,8 @@ public class AuthenticationController {
 
     @PostMapping("/register")
     public ResponseEntity<String> registerUser(@RequestBody @NotNull UserRegistrationDTO registrationDto){
-        if (userService.getUserByEmail(registrationDto.email()).isEmpty() || userService.getUserByUsername(registrationDto.username()).isEmpty()){ // Need to move this logic to the user service
+        // which user service? :P sorry for breaking your changes brah
```

Comment:
You add it to the service then :P Ensure no unique input given exists before saving the user to avoid errors on the database side. Thanks :*

Reply:
this is authentication related though, not my job

keep yourself safe <3